terraform/environments/electronic-monitoring-data/dms_data.tf

```diff
@@ -0,0 +1,14 @@
+data "aws_s3_bucket" "existing_dms_bucket" {
```

Comment:
this wont pass in the production environment - youll need to create a custom one

Reply:
fix applied.